installer/helm/chart/volcano/values.yaml

```diff
@@ -25,6 +25,7 @@ custom:
   leader_elect_enable: false
   enabled_admissions: "/jobs/mutate,/jobs/validate,/podgroups/mutate,/pods/validate,/pods/mutate,/queues/mutate,/queues/validate"
   colocation_enable: false
+  feature_gates: DynamicResourceAllocation=true
```

Comment:
Do we need to enable this feature-gate by default?

Reply:
What do you think? Do we need to enable it by default? I think that if users need to enable dra plugin, they also need to reconfigure the scheduler plugins configmap to enable it, it also looks like featuregate. If we don't enable this feture-gate, users need to do reconfiguring the scheduler plugins configmap and enabling the feature-gate two steps.

Reply:
OK we need not to enable this feature-gate by default, I will add some informers in schedulerCache, some users may need to use k8s below v1.31, if there are no APIs like `DeviceClass`, `ResourceClaim`, etc registered, it will cause some errors.